apps/documentation/package.json

```diff
@@ -3,23 +3,21 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "build": "next build",
+    "build": "yarn g:turbo run build:cmd --filter=documentation...",
+    "build:cmd": "next build",
```

Comment:
`build:cmd` is what turborepo looks for, `build` is just for developer convenience to build all the dependencies that haven't been built yet first